lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -212,6 +207,19 @@ div.course-wrapper {
           }
         }
       }
+
+      .vert > .xblock-student_view {
+        @extend .clearfix;
+        border-bottom: 1px solid #ddd;
+        margin-bottom: ($baseline*0.75);
+        padding: 0 0 15px;
+      }
+
+      .vert > .xblock-student_view.is-hidden {
+          display: none;
+          border-bottom: 0px;
+          margin-bottom: 0px;
```

Comment:
@marcotuts Does this do the trick? I don't think it is possible to have `is-hidden` further up the tree in a clean way.